# Paper for AI4S Learning



# Top Conference in 2026

# Top Conference in 2025
## NeurIPS 2025
[NeurIPS 2025] Scaling Unlocks Broader Generation and Deeper Functional Understanding of Proteins

[NeurIPS 2025] ProtInvTree: Deliberate Protein Inverse Folding with Reward-guided Tree Search

[NeurIPS 2025] DualMPNN: Harnessing Structural Alignments for High-Recovery Inverse Protein Folding

[NeurIPS 2025] MAXWELL: Efficient Learning of Protein-Mutation Stability Landscapes using Protein Language Models

[NeurIPS 2025] Protein Design with Dynamic Protein Vocabulary

[NeurIPS 2025] DecoyDB: A Dataset for Graph Contrastive Learning in Protein-Ligand Binding Affinity Prediction

[NeurIPS 2025] Securing the Language of Life: Inheritable Watermarks from DNA Language Models to Proteins

[NeurIPS 2025] Simultaneous Modeling of Protein Conformation and Dynamics via Autoregression

[NeurIPS 2025] CPSea: Large-scale cyclic peptide-protein complex dataset for machine learning in cyclic peptide design

[NeurIPS 2025] Prot2Text-V2: Protein Function Prediction with Multimodal Contrastive Alignment

[NeurIPS 2025] Towards precision protein-ligand affinity prediction benchmark: A Complete, Modification-Aware DAVIS Dataset

[NeurIPS 2025] Understanding protein function with a multimodal retrieval-augmented foundation model

[NeurIPS 2025] From Likelihood to Fitness: Improving Variant Effect Prediction in Protein and Genome Language Models

[NeurIPS 2025] Rationalized All-Atom Protein Design with Unified Multi-modal Bayesian Flow

[NeurIPS 2025] Generative Modeling of Full-Atom Protein Conformations using Latent Diffusion on Graph Embeddings

[NeurIPS 2025] Estimating protein stability using inverse folding models

[NeurIPS 2025] Equi-mRNA: Protein Translation Equivariant Encoding for mRNA Language Models

[NeurIPS 2025] Repurposing AlphaFold3-like Protein Folding Models for Antibody Sequence and Structure Co-design

[NeurIPS 2025] PSBench: a large-scale benchmark for estimating the accuracy of protein complex structural models

[NeurIPS 2025] Accurately Predicting Protein Mutational Effects via a Hierarchical Many-Body Attention Network

[NeurIPS 2025] Protein Inverse Folding From Structure Feedback

[NeurIPS 2025] ProSpero: Active Learning for Robust Protein Design Beyond Wild-Type Neighborhoods

[NeurIPS 2025] Towards Multiscale Graph-based Protein Learning with Geometric Secondary Structural Motifs

[NeurIPS 2025] ForceFM: Enhancing Protein-Ligand Predictions through Force-Guided Flow Matching

[NeurIPS 2025] Joint Design of Protein Surface and Backbone Using a Diffusion Bridge Model

[NeurIPS 2025] SpecMER: Fast Protein Generation with K-mer Guided Speculative Decoding

[NeurIPS 2025] TEMPO: Temporal Multi-scale Autoregressive Generation of Protein Conformational Ensembles

[NeurIPS 2025] Enhancing Bioactivity Prediction via Spatial Emptiness Representation of Protein-ligand Complex and Union of Multiple Pockets

[NeurIPS 2025] PRING: Rethinking Protein-Protein Interaction Prediction from Pairs to Graphs

[NeurIPS 2025] Learning conformational ensembles of proteins based on backbone geometry

[NeurIPS 2025] Steering Generative Models with Experimental Data for Protein Fitness Optimization

[NeurIPS 2025] Ambient Proteins - Training Diffusion Models on Noisy Structures

[NeurIPS 2025] Latent Retrieval Augmented Generation of Cross-Domain Protein Binders

[NeurIPS 2025] UniZyme: A Unified Protein Cleavage Site Predictor Enhanced with Enzyme Active-Site Knowledge

[NeurIPS 2025] ProteinConformers: Benchmark Dataset for Simulating Protein Conformational Landscape Diversity and Plausibility

## ICML 2025
[ICML 2025] All-atom inverse protein folding through discrete flow matching

[ICML 2025] Aligning Protein Conformation Ensemble Generation with Physical Feedback

[ICML 2025] Boosting Protein Graph Representations through Static-Dynamic Fusion

[ICML 2025] Protriever: End-to-End Differentiable Protein Homology Search for Fitness Prediction

[ICML 2025] ReQFlow: Rectified Quaternion Flow for Efficient and High-Quality Protein Backbone Generation

[ICML 2025] CFP-Gen: Combinatorial Functional Protein Generation via Diffusion Language Models

[ICML 2025] Enhancing Graph Contrastive Learning for Protein Graphs from Perspective of Invariance

[ICML 2025] Diffusion on Language Model Encodings for Protein Sequence Generation

[ICML 2025] A Variational Perspective on Generative Protein Fitness Optimization

[ICML 2025] Sidechain conditioning and modeling for full-atom protein sequence design with FAMPNN

[ICML 2025] Retrieval-Augmented Language Model for Knowledge-aware Protein Encoding

[ICML 2025] An All-Atom Generative Model for Designing Protein Complexes

[ICML 2025] Bridging Protein Sequences and Microscopy Images with Unified Diffusion Models

[ICML 2025] Protein Structure Tokenization: Benchmarking and New Recipe

[ICML 2025] Elucidating the Design Space of Multimodal Protein Language Models

[ICML 2025] P(all-atom) Is Unlocking New Path For Protein Design

[ICML 2025] Flexibility-conditioned protein structure design with flow matching

[ICML 2025] Reward-Guided Iterative Refinement in Diffusion Models at Test-Time with Applications to Protein and DNA Design

[ICML 2025] PPDiff: Diffusing in Hybrid Sequence-Structure Space for Protein-Protein Complex Design

[ICML 2025] From Mechanistic Interpretability to Mechanistic Biology: Training, Evaluating, and Interpreting Sparse Autoencoders on Protein Language Models

[ICML 2025] Steering Protein Language Models

[ICML 2025] Predicting mutational effects on protein binding from folding energy

[ICML 2025] NMA-tune: Generating Highly Designable and Dynamics Aware Protein Backbones

## ICLR 2025
[ICLR 2025] Optimistic Games for Combinatorial Bayesian Optimization with Application to Protein Design

[ICLR 2025] Boltzmann-Aligned Inverse Folding Model as a Predictor of Mutational Effects on Protein-Protein Interactions

[ICLR 2025] Structure Language Models for Protein Conformation Generation

[ICLR 2025] EVA: Geometric Inverse Design for Fast Protein Motif-Scaffolding with Coupled Flow

[ICLR 2025] ProtComposer: Compositional Protein Structure Generation with 3D Ellipsoids

[ICLR 2025] Distilling Structural Representations into Protein Sequence Models

[ICLR 2025] Group Ligands Docking to Protein Pockets

[ICLR 2025] Steering Protein Family Design through Profile Bayesian Flow

[ICLR 2025] AtomSurf: Surface Representation for Learning on Protein Structures

[ICLR 2025] Protein Language Model Fitness is a Matter of Preference

[ICLR 2025] Data Distillation for extrapolative protein design through exact preference optimization

[ICLR 2025] ProtPainter: Draw or Drag Protein via Topology-guided Diffusion

[ICLR 2025] CBGBench: Fill in the Blank of Protein-Molecule Complex Binding Graph

[ICLR 2025] ProteinBench: A Holistic Evaluation of Protein Foundation Models

[ICLR 2025] Integrating Protein Dynamics into Structure-Based Drug Design via Full-Atom Stochastic Flows

[ICLR 2025] Proteina: Scaling Flow-based Protein Structure Generative Models

[ICLR 2025] Learning to engineer protein flexibility

[ICLR 2025] DPLM-2: A Multimodal Diffusion Protein Language Model

[ICLR 2025] Metalic: Meta-Learning In-Context with Protein Language Models

[ICLR 2025] Fast Uncovering of Protein Sequence Diversity from Structure

[ICLR 2025] CheapNet: Cross-attention on Hierarchical representations for Efficient protein-ligand binding Affinity Prediction

[ICLR 2025] Fine-Tuning Discrete Diffusion Models via Reward Optimization with Applications to DNA and Protein Design

[ICLR 2025] Concept Bottleneck Language Models For Protein Design

## AAAI 2025
[AAAI 2025]4D Diffusion for Dynamic Protein Structure Prediction with Reference and Motion Guidance

[AAAI 2025]CryoDomain: Sequence-free Protein Domain Identification from Low-resolution Cryo-EM Density Maps

[AAAI 2025]FoldToken: Learning Protein Language via Vector Quantization and Beyond

[AAAI 2025]HeMeNet: Heterogeneous Multichannel Equivariant Network for Protein Multi-task Learning

[AAAI 2025]CoPRA: Bridging Cross-domain Pretrained Sequence Models with Complex Structures for Protein-RNA Binding Affinity Prediction

[AAAI 2025]Controllable Protein Sequence Generation with LLM Preference Optimization

[AAAI 2025]PScalpel: A Machine Learning-based Guider for Protein Phase-Separating Behaviour Alteration

[AAAI 2025]Accurate Nucleic Acid-Binding Residue Identification Based Domain-Adaptive Protein Language Model and Explainable Geometric Deep Learning

[AAAI 2025]Exploit Your Latents: Coarse-Grained Protein Backmapping with Latent Diffusion Models

[AAAI 2025]Synergy of GFlowNet and Protein Language Model Makes a Diverse Antibody Designer

[AAAI 2025]ProtCLIP: Function-Informed Protein Multi-Modal Learning

## IJCAI 2025
[IJCAI 2025]Enhancing Multimodal Protein Function Prediction Through Dual-Branch Dynamic Selection with Reconstructive Pre-Training 

[IJCAI 2025]PAMol: Pocket-Aware Drug Design Method with Hypergraph Representation of Protein Pocket Structure and Feature Fusion 

[IJCAI 2025]Inferring Causal Protein Signaling Networks with Reinforcement Learning via Artificial Bee Colony Neural Architecture Search

## ICCV 2025
[ICCV 2025]Unsupervised Identification of Protein Compositions and Conformations via Implicit Content-Transformation DisentanglementS

## KDD 2025
[KDD 2025]Energy-Based Models for Predicting Mutational Effects on Proteins

[KDD 2025]Structure-Enhanced Protein Instruction Tuning: Towards General-Purpose Protein Understanding with LLMs

## NAACL2025
[NAACL 2025]GraPPI: A Retrieve-Divide-Solve GraphRAG Framework for Large-scale Protein-protein Interaction Exploration

[NAACL 2025]Protein2Text: Resampling Mechanism to Translate Protein Sequences
## ACL 2025
[ACL 2025]Large Language and Protein Assistant for Protein-Protein Interactions Prediction

[ACL 2025]Enhancing Safe and Controllable Protein Generation via Knowledge Preference Optimization

<!-- ## EMNLP 2025
TBD -->
# Arxiv Paper in 2025

## Submit to ICLR 2026
[Submit to ICLR 2026] ProteinRPN: Towards Accurate Protein Function Prediction with Graph-Based Region Proposals

[Submit to ICLR 2026] ProteinZero: Self-Improving Protein Generation via Online Reinforcement Learning

[Submit to ICLR 2026] Multi-state Protein Design with DynamicMPNN

[Submit to ICLR 2026] Shrinking Proteins with Diffusion

[Submit to ICLR 2026] Design of Ligand-Binding Proteins with Atomic Flow Matching

[Submit to ICLR 2026] Protein Autoregressive Modeling via Multiscale Structure Generation

[Submit to ICLR 2026] Pallatom-Ligand: an All-Atom Diffusion Model for Designing Ligand-Binding Proteins

[Submit to ICLR 2026] Property-Driven Protein Inverse Folding with Multi-Objective Preference Alignment

[Submit to ICLR 2026] La-Proteina: Atomistic Protein Generation via Partially Latent Flow Matching

[Submit to ICLR 2026] ProteinAE: Protein Diffusion Autoencoders for Structure Encoding

[Submit to ICLR 2026] SSProNet: Secondary Structure aware Graph Neural Network for Protein Representation Learning

[Submit to ICLR 2026] MSAFlow: a Unified Approach for MSA Representation, Augmentation, and Family-based Protein Design

[Submit to ICLR 2026] Reverse Distillation: Disentangling and Scaling Protein Language Model Representations

[Submit to ICLR 2026] RadDiff: Retrieval-Augmented Denoising Diffusion for Protein Inverse Folding

[Submit to ICLR 2026] XPro-Design: Rational Protein Engineering Framework Using Explainable AI

[Submit to ICLR 2026] Distilled Protein Backbone Generation

[Submit to ICLR 2026] FractalFold: Towards Fractal Structure Modeling for Hierarchical Inverse Protein Folding

[Submit to ICLR 2026] Greater than the Sum of Its Parts: Building Substructure into Protein Encoding Models

[Submit to ICLR 2026] ThermoRL: Structure-Aware RL for Protein Mutation Design to Enhance Thermostability

[Submit to ICLR 2026] SimpleDesign - A Joint Model for Protein Sequence and Structure Codesign

[Submit to ICLR 2026] PANDORA: Diffusion-based Protein Conformation Generation

[Submit to ICLR 2026] RigidSSL: Rigidity-based Geometric Pretraining for Protein Generation

[Submit to ICLR 2026] SAIR: Enabling Deep Learning for Protein-Ligand Interactions with a Synthetic Structural Dataset

[Submit to ICLR 2026] AutoFold: Ultra-fast Protein Generation via Autoregressive Contact Graph Generation

[Submit to ICLR 2026] PFMBench: Protein Foundation Model Benchmark

[Submit to ICLR 2026] ChiFlow: Torsional Asymmetry Flow Matching For Chirality-Aware Protein Backbone Generation

[Submit to ICLR 2026] Let Physics Guide Your Protein Flows: Topology-aware Unfolding and Generation

[Submit to ICLR 2026] RankFlow: Property-aware Transport for Protein Optimization

[Submit to ICLR 2026] Evolutionary Profiles for Protein Fitness Prediction

[Submit to ICLR 2026] InstructPro: Natural Language Guided Ligand-Binding Protein Design

[Submit to ICLR 2026] Cross-Species Protein Interaction Prediction by Hierarchical Contrastive Pretraining

[Submit to ICLR 2026] ProteinVista: A compute-efficient atom-level 3D CNN that outperforms sequence transformers in protein–ligand prediction

[Submit to ICLR 2026] One protein is all you need

[Submit to ICLR 2026] Attending on Multilevel Structure of Proteins enables Accurate Prediction of Cold-Start Drug-Target Interactions

[Submit to ICLR 2026] Efficient Prediction of Large Protein Complexes via Subunit-Guided Hierarchical Refinement

[Submit to ICLR 2026] RoSE: Enhancing SE(3)-based Protein Backbone Generation via Robust Score Estimation

[Submit to ICLR 2026] SafeProtein: Red-Teaming Framework and Benchmark for Protein Foundation Models

[Submit to ICLR 2026] Speak to a Protein: An Interactive Multimodal Co-Scientist for Protein Analysis

[Submit to ICLR 2026] Structure-Aligned Protein Language Model

[Submit to ICLR 2026] Evolution-Aware Positive-Unlabeled Learning for Protein Design

[Submit to ICLR 2026] BioBlobs: Differentiable Graph Partitioning for Protein Representation Learning

[Submit to ICLR 2026] MolChord: Structure–Sequence Alignment for Protein-Guided Drug Design

[Submit to ICLR 2026] PIPA: An Agent for Protein Interaction Identification and Perturbation Analysis

[Submit to ICLR 2026] KDPRA: A Dual-Molecule Knowledge Distillation model with Cross-Attention Fusion for Protein–RNA Binding Affinity Prediction

[Submit to ICLR 2026] Token-Level Guided Discrete Diffusion for Membrane Protein Design

[Submit to ICLR 2026] Towards Understanding the Shape of Representations in Protein Language Models

[Submit to ICLR 2026] CoPeP: Benchmarking Continual Pretraining for Protein Language Models

[Submit to ICLR 2026] Fast Proteome-Scale Protein Interaction Retrieval via Residue-Level Factorization

[Submit to ICLR 2026] Learning Collective Variables from BioEmu with Time-Lagged Generation
[Submit to ICLR 2026] FlexProtein: Joint Sequence and Structure Pretraining for Protein Modeling

[Submit to ICLR 2026] Geometric Self-Supervised Pretraining on 3D Protein Structures using Subgraphs

[Submit to ICLR 2026] PROTDYN: A FOUNDATION PROTEIN LANGUAGE MODEL FOR THERMODYNAMICS AND DYNAMICS GENERATION

[Submit to ICLR 2026] Protein generation with embedding learning for motif diversification


[Submit to ICLR 2026] Controlling Repetition in Protein Language Models

[Submit to ICLR 2026] RAMM: Robust Adversarial Multimodal Learning for Protein Stability Prediction

[Submit to ICLR 2026] Fast and Interpretable Protein Substructure Alignment via Optimal Transport

[Submit to ICLR 2026] Leveraging Discrete Function Decomposability for Scientific Design

[Submit to ICLR 2026] Lightweight MSA Design Advances Protein Folding From Evolutionary Embeddings

[Submit to ICLR 2026] Constrained Diffusion for Protein Design with Hard Structural Constraints

[Submit to ICLR 2026] PRISM: Enhancing PRotein Inverse Folding through Fine- Grained Retrieval on Structure-Sequence Multimodal Representations

[Submit to ICLR 2026] Boosting In-Silicon Directed Evolution with Fine-Tuned Protein Language Model and Tree Search

[Submit to ICLR 2026] From Supervision to Exploration: What Does Protein Language Model Learn During Reinforcement Learning?

[Submit to ICLR 2026] Drug-ProGO: A Gene Ontology-Enhanced Contrastive Learning Framework for Drug Virtual Screening with Multi-modality Whole-Protein Input

[Submit to ICLR 2026] Mutation2Text: A Unified Protein and Text Language Model for Explaining Mutation Effects

[Submit to ICLR 2026] Improving Protein Sequence Design through Designability Preference Optimization

[Submit to ICLR 2026] PETIMOT: A Novel Framework for Inferring Protein Motions from Sparse Data Using SE(3)-Equivariant Graph Neural Networks

[Submit to ICLR 2026] ESMfluc: Predicting Flexible Regions in a Protein Using Language Models

[Submit to ICLR 2026] Riemannian Variational Flow Matching for Material and Protein Design

[Submit to ICLR 2026] Scalable Generative Modeling of Protein Ligand Trajectories via Graph Neural Diffusion Networks

[Submit to ICLR 2026] GCP-VQVAE: A Geometry-Complete Language for Protein 3D Structure

[Submit to ICLR 2026] GeomMotif: A Benchmark for Arbitrary Geometric Preservation in Protein Generation

[Submit to ICLR 2026] Protein Structure Tokenization via Geometric Byte Pair Encoding

[Submit to ICLR 2026] Empowering Protein Language Model for Sequence-Structure Co-Generation with Continuous Structure Tokens

[Submit to ICLR 2026] DISPROTBENCH: A Disorder-Aware, Task-Rich Benchmark for Evaluating Protein Structure Prediction in Realistic Biological Contexts

[Submit to ICLR 2026] Scaling Atomistic Protein Binder Design with Generative Pretraining and Test-Time Compute

[Submit to ICLR 2026] LiveProteinBench: A Contamination-Free Benchmark for Assessing Models' Specialized Capabilities in Protein Science

[Submit to ICLR 2026] TopoScorer: a light, interpretable predictor for protein-protein binding affinity

[Submit to ICLR 2026] SimpleFold: Folding Proteins is Simpler than You Think

[Submit to ICLR 2026] VenusX: Unlocking Fine-Grained Functional Understanding of Proteins

[Submit to ICLR 2026] Protein-SE(3): Unified Framework and Comprehensive Benchmark for SE(3)-based Protein Structure Design

[Submit to ICLR 2026] UNSUPERVISED MEMBRANE SUBTRACTION IN CRYOGENIC ELECTRON MICROSCOPY IMAGES

[Submit to ICLR 2026] A diffusion model on toric varieties with application to protein loop modeling

[Submit to ICLR 2026] Flow Autoencoders are Effective Protein Tokenizers

[Submit to ICLR 2026] Protein as a Second Language for LLMs

[Submit to ICLR 2026] Advancing Drug-Target Interaction Prediction via Graph Transformers and Residual Protein Embeddings

[Submit to ICLR 2026] Symmetric Dual-Path Integration for Protein Inverse Folding

[Submit to ICLR 2026] SIDGEN - Structure-Informed Diffusion for Generative Modeling of Ligands for Proteins

[Submit to ICLR 2026] Latent Space Codon Optimization Maximizes Protein Expression

[Submit to ICLR 2026] Learning residue level protein dynamics with multiscale Gaussians

[Submit to ICLR 2026] Representing local protein environments with machine learning force fields

[Submit to ICLR 2026] ProtFunAgent: Agentic LLM Cascades for Low-Resource Protein Function Gap-Filling via Homology RAG and Ontology-Constrained Decoding

[Submit to ICLR 2026] Prot2Token: A Unified Framework for Protein Modeling via Next-Token Prediction

[Submit to ICLR 2026] Protap: A Benchmark for Protein Modeling on Realistic Downstream Applications

[Submit to ICLR 2026] Beyond Geometry: Functionally Grounded Molecule Generation for Structure-Based Drug Design

[Submit to ICLR 2026] InterfaceDiff: Interface-Aware Sequence-Structure Co-Design of Protein Complexes with Graph-Based Diffusion

[Submit to ICLR 2026] DCFold: Efficient Protein Structure Generation with Single Forward Pass

[Submit to ICLR 2026] Bridging Protein Structure to Sequence via Local Structure for Inverse Folding

[Submit to ICLR 2026] KGOT: Unified Knowledge Graph and Optimal Transport Pseudo-Labeling for Molecule-Protein Interaction Prediction

[Submit to ICLR 2026] QAProt: Enabling Sequence-to-Text Protein Function Learning with a Comprehensive QA Corpus

[Submit to ICLR 2026] EnerBridge-DPO: Energy-Guided Protein Inverse Folding with Markov Bridges and Direct Preference Optimization

[Submit to ICLR 2026] Prot2RNA: A Diffusion Language Model for Protein-Conditioned mRNA Coding Sequence Generation

[Submit to ICLR 2026] GDEGAN: Gaussian Dynamic Equivariant Graph Attention Network for Ligand Binding Site Prediction

[Submit to ICLR 2026] SLAE: Strictly Local All-atom Environment for Protein Representation

[Submit to ICLR 2026] Learning Early Treatment Strategy from Snapshots for mRNA-Protein Regulatory Networks

[Submit to ICLR 2026] ProtoBind-Diff: Protein-Conditioned Discrete Diffusion for Structure-Free Ligand Generation

[Submit to ICLR 2026] Scalable Spatio-Temporal SE(3) Diffusion for Long-Horizon Protein Dynamics

[Submit to ICLR 2026] InstructPLM-mu: 1-Hour Fine-Tuning of ESM2 Beats ESM3 in Protein Mutation Predictions

[Submit to ICLR 2026] Toward Protein Docking-oriented De Novo Ligand Design via Gradient Inversion

[Submit to ICLR 2026] PDFBench: A Benchmark for De novo Protein Design from Function

[Submit to ICLR 2026] DockedAC: A Dataset with Comprehensive 3D Protein-ligand Complexes for Activity Cliff Analysis

[Submit to ICLR 2026] Performance vs interpretability trade-off of hand-crafted and language model features: The case of protein superfamily classification

[Submit to ICLR 2026] PoseX: AI Defeats Physics-based Methods on Protein Ligand Cross-Docking

[Submit to ICLR 2026] Improving Active-Learning Evaluation, with Applications to Protein-Property Prediction

[Submit to ICLR 2026] Protein Language Model–Aligned Spectra Embeddings for De Novo Peptide Sequencing

[Submit to ICLR 2026] MotifScreen: Generalizing Virtual Screening through Learning Protein-Ligand Interaction Principles

[Submit to ICLR 2026] CAPSUL: A Comprehensive Human Protein Benchmark for Subcellular Localization

[Submit to ICLR 2026] PEPTRIX: A FRAMEWORK FOR EXPLAINABLE PEPTIDE ANALYSIS THROUGH PROTEIN LANGUAGE MODELS

[Submit to ICLR 2026] Mitigating Class Imbalance in Graph-Structured Data via Hierarchical Learning: Insights from Protein Binding Site Prediction

[Submit to ICLR 2026] Beyond Ensembles: Simulating All-Atom Protein Dynamics in a Learned Latent Space


[Submit to ICLR 2026] IgFlow-LM: De Novo Antibody Design via Joint Flow Matching on SE(3) and Protein Language Models Probability Flows