# Paper for AI4S Learning



# Top Conference in 2026
# Top Conference in 2025
## NeurIPS 2025


[NeurIPS 2025] 3D-GSRD: 3D Molecular Graph Auto-Encoder with Selective Re-mask Decoding

[NeurIPS 2025] Accurately Predicting Protein Mutational Effects via a Hierarchical Many-Body Attention Network

[NeurIPS 2025] AlphaFold Database Debiasing for Robust Inverse Folding

[NeurIPS 2025] Ambient Proteins - Training Diffusion Models on Noisy Structures

[NeurIPS 2025] BioReason: Incentivizing Multimodal Biological Reasoning within a DNA-LLM Model

[NeurIPS 2025] CPSea: Large-scale cyclic peptide-protein complex dataset for machine learning in cyclic peptide design

[NeurIPS 2025] DecoyDB: A Dataset for Graph Contrastive Learning in Protein-Ligand Binding Affinity Prediction

[NeurIPS 2025] DualMPNN: Harnessing Structural Alignments for High-Recovery Inverse Protein Folding

[NeurIPS 2025] EnzyControl: Adding Functional and Substrate-Specific Control for Enzyme Backbone Generation

[NeurIPS 2025] Equi-mRNA: Protein Translation Equivariant Encoding for mRNA Language Models

[NeurIPS 2025] From Likelihood to Fitness: Improving Variant Effect Prediction in Protein and Genome Language Models

[NeurIPS 2025] Generative Modeling of Full-Atom Protein Conformations using Latent Diffusion on Graph Embeddings

[NeurIPS 2025] Joint Design of Protein Surface and Backbone Using a Diffusion Bridge Model

[NeurIPS 2025] Latent Retrieval Augmented Generation of Cross-Domain Protein Binders

[NeurIPS 2025] MAXWELL: Efficient Learning of Protein-Mutation Stability Landscapes using Protein Language Models

[NeurIPS 2025] Mol-LLaMA: Towards General Understanding of Molecules in Large Molecular Language Model

[NeurIPS 2025] NeuralPLexer3: Accurate Biomolecular Complex Structure Prediction with Flow Models

[NeurIPS 2025] Prot2Text-V2: Protein Function Prediction with Multimodal Contrastive Alignment

[NeurIPS 2025] ProteinConformers: Benchmark Dataset for Simulating Protein Conformational Landscape Diversity and Plausibility

[NeurIPS 2025] Protein Design with Dynamic Protein Vocabulary

[NeurIPS 2025] Protein Inverse Folding From Structure Feedback

[NeurIPS 2025] ProtInvTree: Deliberate Protein Inverse Folding with Reward-guided Tree Search

[NeurIPS 2025] PSBench: a large-scale benchmark for estimating the accuracy of protein complex structural models

[NeurIPS 2025] Repurposing AlphaFold3-like Protein Folding Models for Antibody Sequence and Structure Co-design

[NeurIPS 2025] Scaling Unlocks Broader Generation and Deeper Functional Understanding of Proteins

[NeurIPS 2025] Unified all-atom molecule generation with neural fields

[NeurIPS 2025] Understanding protein function with a multimodal retrieval-augmented foundation model

[NeurIPS 2025] UniSite: The First Cross-Structure Dataset and Learning Framework for End-to-End Ligand Binding Site Detection

[NeurIPS 2025] UniZyme: A Unified Protein Cleavage Site Predictor Enhanced with Enzyme Active-Site Knowledge

## ICML 2025
[ICML 2025] All-atom inverse protein folding through discrete flow matching

[ICML 2025] Aligning Protein Conformation Ensemble Generation with Physical Feedback

[ICML 2025] Boosting Protein Graph Representations through Static-Dynamic Fusion

[ICML 2025] Protriever: End-to-End Differentiable Protein Homology Search for Fitness Prediction

[ICML 2025] ReQFlow: Rectified Quaternion Flow for Efficient and High-Quality Protein Backbone Generation

[ICML 2025] CFP-Gen: Combinatorial Functional Protein Generation via Diffusion Language Models

[ICML 2025] Enhancing Graph Contrastive Learning for Protein Graphs from Perspective of Invariance

[ICML 2025] Diffusion on Language Model Encodings for Protein Sequence Generation

[ICML 2025] A Variational Perspective on Generative Protein Fitness Optimization

[ICML 2025] Sidechain conditioning and modeling for full-atom protein sequence design with FAMPNN

[ICML 2025] Retrieval-Augmented Language Model for Knowledge-aware Protein Encoding

[ICML 2025] An All-Atom Generative Model for Designing Protein Complexes

[ICML 2025] Bridging Protein Sequences and Microscopy Images with Unified Diffusion Models

[ICML 2025] Protein Structure Tokenization: Benchmarking and New Recipe

[ICML 2025] Elucidating the Design Space of Multimodal Protein Language Models

[ICML 2025] P(all-atom) Is Unlocking New Path For Protein Design

[ICML 2025] Flexibility-conditioned protein structure design with flow matching

[ICML 2025] Reward-Guided Iterative Refinement in Diffusion Models at Test-Time with Applications to Protein and DNA Design

[ICML 2025] PPDiff: Diffusing in Hybrid Sequence-Structure Space for Protein-Protein Complex Design

[ICML 2025] From Mechanistic Interpretability to Mechanistic Biology: Training, Evaluating, and Interpreting Sparse Autoencoders on Protein Language Models

[ICML 2025] Steering Protein Language Models

[ICML 2025] Predicting mutational effects on protein binding from folding energy

[ICML 2025] NMA-tune: Generating Highly Designable and Dynamics Aware Protein Backbones

## ICLR 2025
[ICLR 2025] Optimistic Games for Combinatorial Bayesian Optimization with Application to Protein Design

[ICLR 2025] Boltzmann-Aligned Inverse Folding Model as a Predictor of Mutational Effects on Protein-Protein Interactions

[ICLR 2025] Structure Language Models for Protein Conformation Generation

[ICLR 2025] EVA: Geometric Inverse Design for Fast Protein Motif-Scaffolding with Coupled Flow

[ICLR 2025] ProtComposer: Compositional Protein Structure Generation with 3D Ellipsoids

[ICLR 2025] Distilling Structural Representations into Protein Sequence Models

[ICLR 2025] Group Ligands Docking to Protein Pockets

[ICLR 2025] Steering Protein Family Design through Profile Bayesian Flow

[ICLR 2025] AtomSurf: Surface Representation for Learning on Protein Structures

[ICLR 2025] Protein Language Model Fitness is a Matter of Preference

[ICLR 2025] Data Distillation for extrapolative protein design through exact preference optimization

[ICLR 2025] ProtPainter: Draw or Drag Protein via Topology-guided Diffusion

[ICLR 2025] CBGBench: Fill in the Blank of Protein-Molecule Complex Binding Graph

[ICLR 2025] ProteinBench: A Holistic Evaluation of Protein Foundation Models

[ICLR 2025] Integrating Protein Dynamics into Structure-Based Drug Design via Full-Atom Stochastic Flows

[ICLR 2025] Proteina: Scaling Flow-based Protein Structure Generative Models

[ICLR 2025] Learning to engineer protein flexibility

[ICLR 2025] DPLM-2: A Multimodal Diffusion Protein Language Model

[ICLR 2025] Metalic: Meta-Learning In-Context with Protein Language Models

[ICLR 2025] Fast Uncovering of Protein Sequence Diversity from Structure

[ICLR 2025] CheapNet: Cross-attention on Hierarchical representations for Efficient protein-ligand binding Affinity Prediction

[ICLR 2025] Fine-Tuning Discrete Diffusion Models via Reward Optimization with Applications to DNA and Protein Design

[ICLR 2025] Concept Bottleneck Language Models For Protein Design

## AAAI 2025
[AAAI 2025]4D Diffusion for Dynamic Protein Structure Prediction with Reference and Motion Guidance

[AAAI 2025]CryoDomain: Sequence-free Protein Domain Identification from Low-resolution Cryo-EM Density Maps

[AAAI 2025]FoldToken: Learning Protein Language via Vector Quantization and Beyond

[AAAI 2025]HeMeNet: Heterogeneous Multichannel Equivariant Network for Protein Multi-task Learning

[AAAI 2025]CoPRA: Bridging Cross-domain Pretrained Sequence Models with Complex Structures for Protein-RNA Binding Affinity Prediction

[AAAI 2025]Controllable Protein Sequence Generation with LLM Preference Optimization

[AAAI 2025]PScalpel: A Machine Learning-based Guider for Protein Phase-Separating Behaviour Alteration

[AAAI 2025]Accurate Nucleic Acid-Binding Residue Identification Based Domain-Adaptive Protein Language Model and Explainable Geometric Deep Learning

[AAAI 2025]Exploit Your Latents: Coarse-Grained Protein Backmapping with Latent Diffusion Models

[AAAI 2025]Synergy of GFlowNet and Protein Language Model Makes a Diverse Antibody Designer

[AAAI 2025]ProtCLIP: Function-Informed Protein Multi-Modal Learning

## IJCAI 2025
[IJCAI 2025]Enhancing Multimodal Protein Function Prediction Through Dual-Branch Dynamic Selection with Reconstructive Pre-Training 

[IJCAI 2025]PAMol: Pocket-Aware Drug Design Method with Hypergraph Representation of Protein Pocket Structure and Feature Fusion 

[IJCAI 2025]Inferring Causal Protein Signaling Networks with Reinforcement Learning via Artificial Bee Colony Neural Architecture Search

## ICCV 2025
[ICCV 2025]Unsupervised Identification of Protein Compositions and Conformations via Implicit Content-Transformation DisentanglementS

## KDD 2025
[KDD 2025]Energy-Based Models for Predicting Mutational Effects on Proteins

[KDD 2025]Structure-Enhanced Protein Instruction Tuning: Towards General-Purpose Protein Understanding with LLMs

## NAACL2025
[NAACL 2025]GraPPI: A Retrieve-Divide-Solve GraphRAG Framework for Large-scale Protein-protein Interaction Exploration

[NAACL 2025]Protein2Text: Resampling Mechanism to Translate Protein Sequences
## ACL 2025
[ACL 2025]Large Language and Protein Assistant for Protein-Protein Interactions Prediction

[ACL 2025]Enhancing Safe and Controllable Protein Generation via Knowledge Preference Optimization

<!-- ## EMNLP 2025
TBD -->
# Arxiv Paper in 2026
# Arxiv Paper in 2025